### DISCUSSION TITLE
Copilot in Jetbrains Rider(V2021.3.4) , Seems like Copilot do not work after lines over about 95 to 100 in a .cs file

### DISCUSSION THREAD

**Discussion by FANG-Zhiguo** [community/community#14641] [Copilot Conversations] [answered]
Copilot in Jetbrains Rider(V2021.3.4) , Seems like Copilot do not work after lines over about 95 to 100 in a .cs file

**Comment by jansorg** [comment, marked as answer]
This shouldn't make a difference.

Suggestions depend on the current context, sometimes there's just nothing to suggest. Try with a new function or method after these lines to test if suggestions are still returned, for example.
If VSCode+Copilot is showing suggestions for the same position and file, then there's something wrong with the JetBrains plugin.

**Reply by FANG-Zhiguo** [reply]
Thank you for answering my question.
open this file with vscode, the copilot can display suggestions.
Should I go to the JetBrains plugin market to feed back this question?

**Reply by jansorg** [reply]
Thanks! Reporting there is not needed, here's the best place for this issue.
1. If the code is public, could you post a screenshot of what VSCode is suggesting for your context?
2. Which version of the JetBrains plugin are you using?

**Reply by FANG-Zhiguo** [reply]
1.I repeated this issue many times.
When the content length of a single file exceeds 3200,
If the input position is after 3200 (± 100), copilot will not display any suggestions,
However, if the input position is within 3200 (± 100), copilot can still display suggestions.
![image](https://user-images.githubusercontent.com/39255754/162760243-3be5a053-5aa5-4063-a17b-be131daee2f0.png)
![image](https://user-images.githubusercontent.com/39255754/162760393-7748d5da-8c7c-46e4-b1b8-d5df8a3fe338.png)

2.
Jetbrains Rider version: 2021.3.4,
GitHub Copilot plugin version: 1.1.15.1115 .

**Reply by jansorg** [reply]
Thanks! We've found a problem and fixed it for the next update. Please let us know if the next update is still not showing suggestions for your code.
